### PR TITLE
feat(failurechecker): track multiple failures and autoclose after success

### PR DESCRIPTION
**Pull request by chingor13**
This PR does a few things:

* track multiple release PR failures (shown as a list)
* updates the issue body when a new release fails
* ignores release PRs that are marked as both `autorelease: failed` and `autorelease: published` (in case you retry and it succeeds)
* closes the open issue when release failures are resolved

### CODE REVIEW

File: packages/failurechecker/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "start": "probot run ./build/src/failurechecker.js",
     "start:local": "node ./build/src/local.js",
     "pretest": "npm run compile",
-    "test": "cross-env LOG_LEVEL=fatal c8 mocha build/test",
+    "test": "cross-env LOG_LEVEL=fatal c8 mocha --exit build/test",
```
Comment (JustinBeckwith) [inline packages/failurechecker/package.json:25]: _grumble_ usually you only need that when a web-server connection isn't getting cleaned up.  I look at that as code smell.

**Review by JustinBeckwith — APPROVED** [review]

File: packages/failurechecker/package.json (hunk 1 above)
Comment (chingor13) [reply packages/failurechecker/package.json:25]: That wasn't meant to stay. While writing the tests, probot is retrying api requests that fail (unlocked) with backoff and I temporarily bumped the timeout to make them appear. I'll revert this part

Comment (JustinBeckwith) [reply packages/failurechecker/package.json:25]: I have a fix for this coming

Comment (chingor13) [reply packages/failurechecker/package.json]: This is now removed.

**Pull request merged**